app/assets/stylesheets/styles/components/guides/_guides.css.scss

```diff
@@ -189,73 +360,60 @@
   }
 }
 
-
 .show-guide {
 
-  .canopy{
-    height: 300px;
-    margin-bottom: 1rem;
-  }
-
-  .title{
-    .guide-name {
-      text-transform: capitalize;
-    }
-
-    .by-line {
-      font-size: 1rem;
-      color: #666666;
-    }
+  background-color: #f6f6f6;
 
-    .guide-controls {
-      float: right;
-      text-align: right;
-    }
+  .compatibility-circle-container {
+    margin-top: 1rem;
+    margin-bottom: 1rem;
+    margin-right: auto;
+    margin-left: auto;
+    width: 50px;
   }
 
   .requirements {
-    .box {
-      padding: 0;
-
-      table {
-        width: 100%;
-        height: 100%;
-        margin: 0;
-
-        tr {
+    padding: 0;
 
-          &.low {
-            background: $compatibility-low-color-bg;
-          }
+    table {
+      width: 100%;
+      height: 100%;
+      margin: 0;
+      border: none;
 
-          &.medium {
-            background: $compatibility-med-color-bg;
-          }
+      tr {
 
-          &.high {
-            background: $compatibility-high-color-bg;
-          }
+        &.low {
+          background: $compatibility-low-color-bg;
+        }
 
-          td {
-            padding: 16px;
-          }
+        &.medium {
+          background: $compatibility-med-color-bg;
         }
 
-        p {
-          margin: 0;
+        &.high {
+          background: $compatibility-high-color-bg;
         }
 
-        .basic-need-name {
-          span {
-            padding-left: 10px;
-            font-weight: 600;
-          }
+        td {
```

Comment:
Hound is right, but I'm going to be fixing this kind of stuff in a SCSS refactor upcoming. This isn't because of stuff you're writing, just the way our code is organized at the moment.